src/assets/Generator.Shared/Utf8JsonWriterExtensions.cs

```diff
@@ -140,5 +140,33 @@ public static void WriteObjectValue(this Utf8JsonWriter writer, object value)
                     throw new NotSupportedException("Not supported type " + value.GetType());
             }
         }
+
+        public static void WriteBinaryDataValue(this Utf8JsonWriter writer, BinaryData value)
+        {
+            string inputValue = value.ToString();
```

Comment:
I don't like that we need to create a copy of the `BinaryData` content. Can we cast `BinaryData` to `Span<byte>` and then use [UTF8Encoding.GetChars](https://docs.microsoft.com/en-us/dotnet/api/system.text.utf8encoding.getchars?view=net-6.0)?

Reply:
Or maybe even use `Utf8JsonReader` to validate the span?

Reply:
I change it to ReadOnlySpan<char> using `GetChars`.